dbus-opendtu.py

```diff
@@ -306,6 +310,12 @@ def _refreshData(self):
       logging.info("Converting response to JSON failed")
       raise ValueError("Converting response to JSON failed")
 
+    if self.dtuvariant == 'opendtu':
+      if not 'AC' in meter_data['inverters'][self.pvinverternumber]:
+        logging.info("You do not have the latest OpenDTU Version to run this scrip, please upgrade your OpenDTU to at least version 4.4.3")
```

Comment:
typo: scrip_t
Ausprobieren kannst du das ja vermutlich nicht, oder?

Reply:
typo is raus

getestet habe ich damit, dass ich anstatt AC einmal ACT hatte und dann ist das Script mit dem Raise rausgegangen.